manifests/kustomize/third-party/argo/upstream/manifests/namespace-install/kustomization.yaml

```diff
@@ -4,16 +4,16 @@ resources:
   - ../base
   - ./argo-server-rbac
   - ./workflow-controller-rbac
-patches:
```

Comment:
this was just updated by https://github.com/kubeflow/pipelines/commit/af085769fbb29a72d1b18c6a47d474e58dbd5577, but these manifests are pulled directly from upstream.  It appears upstream itself updated its manifests to using `patches` in the very next release (v3.5.0), so I can revert these specific changes back to use `patches` (and therefore we will maintain kustomize5 compatibility), or just allow the reversion to happen again at next argo upgrade